certified-connectors/bitskout/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@saferoombiz Please fix the indentation. All files should be properly formatted.